readme.md

```diff
@@ -93,6 +93,13 @@ Default: `-1` *(means no limit)*
 
 Number of warnings to trigger non-zero exit code.
 
+### stdout
```

Comment:
Needs a better name. Something that makes it clear it's in a addition to writing the file. This makes it seem that it won't write to stdout at all, even when `outputFile` is not set.